content/millennium/r4/definitional-artifacts/questionnaire.md

```diff
@@ -9,7 +9,7 @@ title: Questionnaire | R4 API
 
 ## Overview
 
-The Questionnaire resource defines and organizes questions and the allowed answers to the questions typically used to collect patient healthcare information.  A questionnaire is a snapshot in time and should be retrieved before each use. Examples of questionnaires are forms used to collect a patient's social history or family member history.  Currently, only social history is supported.
+The Questionnaire resource defines and organizes questions and the allowed answers to the questions typically used to collect patient healthcare information.  A questionnaire is a snapshot in time and should be retrieved before each use. Examples of questionnaires are forms used to collect a patient's social history or family member history.  Currently, only social history is supported. To find a questionnaire for a specific patient, first search QuestionnaireResponse by patient. Then take the 'questionnaire' field from the QuestionnaireResponse and use that to retrieve the original questionnaire.
```

Comment:
nit: i like 'in order to find a questionnaire...'
and
'Then _use_ the ...' '...to  retrieve the original...'

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/6b1e581f4f01eb2d8a1a69a8a0f857da78c0fcc3